akatsuki_utils_py/enums/gamemodes.py

```diff
@@ -0,0 +1,12 @@
+from enum import IntEnum
+
+class GameMode(IntEnum):
+    STANDARD = 0
+    TAIKO = 1
+    CTB = 2
+    MANIA = 3
+
+class SubMode(IntEnum):
```

Comment:
there's a preexisting standard in the codebase here, better to leave it for now as to not cause further fragmentation

```suggestion
class AkatsukiMode(IntEnum):
```